### DISCUSSION TITLE
Cannot find module 'mantine-react-table/locales/vi'

### DISCUSSION THREAD

**Discussion by viettbq2002** [KevinVandy/mantine-react-table#244] [Q&A]
Hi, I got this error when trying to import to use localizations, is there any additional install to use?
I Already install MRT v2 and full peer dependency in my project
![image](https://github.com/KevinVandy/mantine-react-table/assets/98259617/3fbc757d-5d2e-4346-8481-019b7d0c8c71)

**Comment by KevinVandy** [comment]
Are you using Remix?

<img width="1203" alt="image" src="https://github.com/KevinVandy/mantine-react-table/assets/28243511/e6b7c70c-7674-43b5-8f7e-57c477659145">

**Reply by KevinVandy** [reply]
also, what version of MRT? Alpha 10?

**Reply by viettbq2002** [reply]
I am not using remix , just vite react with MRT 2.0.0-alpha.9"

**Reply by viettbq2002** [reply]
Thank you
Change to this import worked now `import { MRT_Localization_VI } from "mantine-react-table/locales/vi/index.js";` seem like it not mention in documentation 😁

**Comment by antkit** [comment]
me too. sometimes it can work without this error, but the error will be reported if i reload the page by pressing F5:
```
Cannot find module 'node_modules\mantine-react-table\locales\zh-Hans'
```

and sometimes report this error if i use default locale:
```
Error: @mantine/core: MantineProvider was not found in component tree, make sure you have it in your app
```

i'm using v2.0.0-alpha.10. i have tried both mantine 7.3 and 7.4.

**Reply by KevinVandy** [reply]
Are you using Remix?

**Reply by antkit** [reply]
No, I'm using Next.js 14.0.4.

**Reply by KevinVandy** [reply]
I've been seeing this sometimes too. It's weird. We have some convos in the discord. Personally, I only see it in new Mantine .0 releases. Don't know why
<img width="1309" alt="image" src="https://github.com/KevinVandy/mantine-react-table/assets/28243511/13faad13-edc1-4c05-9322-47381142bf85">
<img width="1069" alt="image" src="https://github.com/KevinVandy/mantine-react-table/assets/28243511/bc3ef088-1732-49e7-a374-735bebc36200">

**Comment by ubunterro** [comment]
I use Vite with react, and such import works for me
`import { MRT_Localization_RU } from 'mantine-react-table/locales/ru/index.cjs'`

**Reply by GRSdzn** [reply]
thank you!
